<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>resForm</class>
 <widget class="QWidget" name="resForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Kunder med aktive tilbud</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QComboBox" name="klist" />
         </item>
         <item>
          <layout class="QHBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLineEdit" name="knr" >
             <property name="enabled" >
              <bool>true</bool>
             </property>
             <property name="frame" >
              <bool>false</bool>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="showNr" >
           <property name="maximumSize" >
            <size>
             <width>85</width>
             <height>24</height>
            </size>
           </property>
           <property name="text" >
            <string>Vis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeForm" >
           <property name="maximumSize" >
            <size>
             <width>85</width>
             <height>24</height>
            </size>
           </property>
           <property name="text" >
            <string>Luk</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tilb_list" >
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
